swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -8247,6 +8467,22 @@ components:
           $ref: "#/components/schemas/tppMessageText"
 
 
+    tppMessage201PaymentInitiation: 
+      type: object
+      required:
+        - category
+        - code
+      properties:
+        category:
+           $ref: "#/components/schemas/tppMessageCategory"
+        code:
+          $ref: "#/components/schemas/MessageCode201PaymentInitiation"
+        path:
+          type: string
+        text:
+          $ref: "#/components/schemas/tppMessageText"
+
+
```

Comment:
Relevant Change: tppMessage201PaymentInitiation is new.
Adoption approach: Follow the change